pallets/gear/src/benchmarking/syscalls.rs

```diff
@@ -1064,6 +1064,69 @@ where
         Self::prepare_handle(code, 0)
     }
 
+    pub fn gr_error(r: u32) -> Result<Exec<T>, &'static str> {
+        let string_offset = 1;
+        let string_len = 100;
+        let error_offset = string_offset + string_len;
+        let error_len_offset = error_offset + string_len;
+
+        let code = WasmModule::<T>::from(ModuleDefinition {
+            memory: Some(ImportedMemory::max::<T>()),
+            imported_functions: vec!["gr_debug", "gr_error"],
```

Comment:
```suggestion
            imported_functions: vec!["gr_error"],
```
That's strange to trigger error using gr_debug
I'd go using `gr_status_code` for handle entry, for example